scripts/release_static

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+
+static_assets_bucket="aws-cms-oit-iusg-spe-easi-dev-easi-dev-static-assets"
```

Comment:
Can you move the bucket name to a CircleCI environment variable? Eventually, this script will be run in multiple environments, so we'll probably need to pull the bucket name from another parameter source, like SSM, but moving it to a CircleCI environment variable would work for now.

Reply:
e2abb62